web/src/compose_ui.js

```diff
@@ -350,6 +354,7 @@ export function handle_keydown(event, $textarea) {
 }
 
 export function handle_keyup(_event, $textarea) {
+    shift_pressed = false;
```

Comment:
This is illogical. Another key could have been released without Shift being released.

Reply:
Oops, so sorry for the oversight; I was figuring out the condition and forgot to amend the commit once I'd got it :grimacing: 

Have fixed this now!